app/commands/testingCentres.js

```diff
@@ -0,0 +1,49 @@
+const request = require('request')
+const prepareAnswer = require('./../utils/prepareAnswer')
+const { prepareError } = require('./../utils/prepareError')
+const resourcesUrl = 'https://api.covid19india.org/resources/resources.json'
+let options = { json: true }
+
+const TESTING_RESOURCE_CATEGORY = "CoVID-19 Testing Lab"
+
+function testingCentres(body, bot, chatId, stateUserMessage) {
+      request(resourcesUrl, options, (error, res, resourcesBody) => {
+            if (error) {
+                  return console.log(error)
+            }
+
+            if (!error && res.statusCode == 200) {
+                  var statewise = body['statewise']
+                  var stateName
```

Comment:
I think you can use for loop here because we can just break out of the loop when we finally find the state, `every` always runs for all the elements and you cannot `break;`.

Reply:
We can break out of an `every` loop, but I've changed it to a `for` loop for consistency

Reply:
Ah, my bad. `every` loop ends when an element fails the test, `for` loop is also acceptable.